### PR TITLE
Configured ad skip button in AdViewController

### PR DESCRIPTION
**Pull request by RomanTysiachnik**
## Changes
- Button has 3 states: 
   - `unavailable` - no skip provided;
   - `awaiting` - ad is skipple after some amount of time;
   - `available` - ad is allowed to be skipped.
- Updated snapshot tests with last two cases

@VerizonAdPlatforms/mobile-sdk-developers: Please review.

### CODE REVIEW

File: PlayerControls/sources/AdVideoControls.swift [hunk 1]
```diff
@@ -64,25 +67,49 @@ public final class AdVideoControls: UIViewController {
...
+            if adSkipButton.isEnabled == false {
+                adSkipButton.isEnabled = true
+            }
+            adSkipButton.isHidden = false
+            adSkipButton.titleLabel?.text = "Skip"
+        case .awaiting(let time):
+            adSkipButton.isHidden = false
+            adSkipButton.isEnabled = false
+            adSkipButton.titleLabel?.alpha = 0.5
+            adSkipButton.titleLabel?.text = "Skip in \(time)"
```
Comment (VladyslavAnokhin) [inline PlayerControls/sources/AdVideoControls.swift]: I'm not sure that you can change the text in such way. You have to use `setTitle("", for: )`

File: PlayerControls/sources/AdVideoControls.swift [hunk 2]
```diff
@@ -64,25 +67,49 @@ public final class AdVideoControls: UIViewController {
...
+        
+        switch props.adSkipState {
+        case .unavailable:
+            adSkipButton.isHidden = true
+        case .available:
+            if adSkipButton.isEnabled == false {
+                adSkipButton.isEnabled = true
+            }
+            adSkipButton.isHidden = false
+            adSkipButton.titleLabel?.text = "Skip"
```
Comment (VladyslavAnokhin) [inline PlayerControls/sources/AdVideoControls.swift]: Same here

File: PlayerControls/sources/AdVideoControls.swift [hunk 3]
```diff
@@ -64,25 +67,49 @@ public final class AdVideoControls: UIViewController {
...
         remainingPlayTimeLabel.text = props.seeker?.remainingPlayTime
         airplayActiveView.isHidden = props.airplayActiveViewHidden
+        
+        adSkipButton.titleLabel?.textAlignment = .center
+        
+        switch props.adSkipState {
+        case .unavailable:
+            adSkipButton.isHidden = true
+        case .available:
+            if adSkipButton.isEnabled == false {
```
Comment (VladyslavAnokhin) [inline PlayerControls/sources/AdVideoControls.swift]: Why do we need this `if`

File: PlayerControls/sources/AdVideoControls.swift (hunk 1 above)
Comment (AndriiMoskvin) [reply PlayerControls/sources/AdVideoControls.swift]: You can't :)

File: PlayerControls/sources/AdVideoControls.swift (hunk 3 above)
Comment (AndriiMoskvin) [reply PlayerControls/sources/AdVideoControls.swift]: Yeah, just make `isEnabled = true` also.

Comment (RomanTysiachnik) [reply PlayerControls/sources/AdVideoControls.swift]: Dropped as it is not needed anymore

File: PlayerControls/sources/AdVideoControls.swift [hunk 4]
```diff
@@ -64,25 +67,47 @@ public final class AdVideoControls: UIViewController {
         seekerView.accessibilityLabel = props.seeker?.accessibilityLabel ?? ""
         remainingPlayTimeLabel.text = props.seeker?.remainingPlayTime
         airplayActiveView.isHidden = props.airplayActiveViewHidden
+        
+        adSkipButton.titleLabel?.textAlignment = .center
```
Comment (AndriiMoskvin) [inline PlayerControls/sources/AdVideoControls.swift]: Why this is not specified in XIB?

Comment (RomanTysiachnik) [reply PlayerControls/sources/AdVideoControls.swift]: Fixed

**Pull request merged**